kfdefs/overlays/moc/smaug/trino/hive-metastores/kustomization.yaml

```diff
@@ -4,5 +4,4 @@ kind: Kustomization
 namespace: odh-trino
 resources:
   - data-science_general
```

Comment:
keep kfdefs/overlays/moc/smaug/trino contents for now, but make a follow up issue to replace the trino morty catalog with a catalog backed by a bucket in smaug

Reply:
I have made this into a follow on issue: https://github.com/operate-first/apps/issues/2352